charts/monitor/charts/grafana/templates/monitor-grafana-pvc.yaml

```diff
@@ -6,7 +6,11 @@ metadata:
   labels:
     heritage: deis
   annotations:
-    volume.beta.kubernetes.io/storage-class: {{ .Values.persistence.storageClass }}
+        {{- if .Values.persistence.storageClass }}
```

Comment:
Could this be dedented two spaces? Sorry, my OCD spidey-sense is triggered by whitespace misalignment.